src/constants/constants.ts

```diff
@@ -34,9 +34,17 @@ export const ERC725Y_INTERFACE_IDS = {
   //     - getData(bytes32[])
   //     - setData(bytes32[],bytes[])
   '3.0': '0x714df77c',
+  // version 5.0.0 removed function overloading
```

Comment:
@Hugoo should we also add version 4?
From what I remember, we mention that we don't want to support old versioning. So it might be worth thinking if we should just delete the old versions mentioned above.

Reply:
Also can't we import the @erc725/smart-contracts package and import the ERC725Y interface ID of v5 from the package?

Can we install @erc725/smart-contracts as a dev dependency to prevent having the package size to be too big, or we can't have it as dev-dependency?

Reply:
@CJ42 In terms of `interfaceId`, version 3 is the same as version 4. That's why it's not added.
For deleting the old versions, at some point, they should be deleted, but I guess this should not be done in this PR nor in the near future unless we reach a stable version that goes to production.

Reply:
@Hugoo Done 🙏